ember-welcome-page/src/components/welcome-page.ts

```diff
@@ -0,0 +1,45 @@
+import { getOwner } from '@ember/application';
+/* @ts-expect-error: Cannot find module '@ember/version' or its corresponding type declarations. */
+import { VERSION } from '@ember/version';
```

Comment:
Currently, there is no package named `@types/ember__version`. I wasn't sure what to do in this case, other than using `@ts-expect-error` to ignore the TypeScript error and indicate to other contributors that we should revisit this line later.

According to `ember-source`, `VERSION` has the type of `string`.

https://github.com/emberjs/ember.js/blob/v4.9.3/packages/@ember/version/type-tests/index.test.ts#L4